src/actions/useCreateAction.ts

```diff
@@ -123,9 +123,13 @@ export default <
       doHandleSuccess(result)
       // @TODO: Pass result, id, config
       afterSuccess(request)
+
+      return Promise.resolve()
     } else {
       // @TODO: Pass result (error response), id, config
       afterFailure(request)
+
+      return Promise.reject(request.errorMessage)
```

Comment:
NOTE:
Please have a look at this comment:
https://github.com/Amsterdam/zaken-frontend/pull/89/files#r448217751